pkg/apiserver/user/sqlauth/sqlauth.go

```diff
@@ -60,3 +147,22 @@ func (a *Authenticator) Authenticate(f user.AuthenticateForm) (*utils.SessionUse
 		IsWriteable:  true,
 	}, nil
 }
+
+// grantRows examples:
+// - GRANT PROCESS,SHOW DATABASES,CONFIG ON *.* TO 'dashboardAdmin'@'%'
+// - GRANT SYSTEM_VARIABLES_ADMIN,RESTRICTED_VARIABLES_ADMIN,RESTRICTED_STATUS_ADMIN,RESTRICTED_TABLES_ADMIN ON *.* TO 'dashboardAdmin'@'%'
```

Comment:
Please also check out what will happen when there are table grants, and role grants.

Reply:
Searched some documents, it seems there are 2 kinds of format to show grants for the current user:

- `GRANT [grants] ON [db.table] TO [user]`
- `GRANT [roles] TO [user]`

@crazycs520 please help confirm.

Reply:
Just as @baurine said, there are only 2 kinds of grant syntax.